google-cloud-storage/acceptance/storage/file_test.rb

```diff
@@ -355,24 +355,25 @@
 
   it "should upload and partially download text" do
     inmemory = StringIO.new "Hello world!"
-    uploaded = bucket.create_file inmemory, "uploaded/with/inmemory.png"
+    uploaded = bucket.create_file inmemory, "uploaded/with/inmemory.txt"
```

Comment:
I wonder if the reason we've seen this test fail occasionally is because the mime-type was being set incorrectly due to the file extension.